calculator-data/src/main/java/camp/nextstep/edu/calculator/data/db/RecordDatabase.kt

```diff
@@ -0,0 +1,35 @@
+package camp.nextstep.edu.calculator.data.db
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import camp.nextstep.edu.calculator.data.model.RecordEntity
+
+
+@Database(
+    entities = [RecordEntity::class],
+    version = 1,
+    exportSchema = false
+)
+abstract class RecordDatabase : RoomDatabase() {
+
+    abstract fun recordDao(): RecordDao
+
+    companion object {
+        @Volatile
+        private var INSTANCE: RecordDatabase? = null
```

Comment:
```suggestion
        private var instance: RecordDatabase? = null
```

상수가 아닌 경우에는 변수명을 소문자로 지정해야합니다!

Reply:
해당 참조값은 Singleton이여서 한번 선언된 뒤에 변경되지 않고  RecordDatabase로 객체를 외부에서 접근하지 못하게 했기때문에 변경할 수 없으니 상수로 봐도 되지 않을까요?

Reply:
좋은 의견이네요!
하지만 var로 선언되어있기때문에 getInstance외에 언제든 다른 함수가 추가된다면 변경될 여지가 있는 코드가 되겠네요..!
따라서 완전히 상수로 볼 수 없다고 생각이 됩니다.

Reply:
그렇군요! 다른 함수로 접근가능하니 상수가 아닐 수 있네용!

Reply:
네네 맞습니다. 다른 함수로 접근 가능하고, 그로인해 변경될 여지가 있기 때문에 상수라고 보기는 어려울 것 같습니다!